tests/trainers/test_tasks.py

```diff
@@ -9,19 +9,31 @@
 from _pytest.monkeypatch import MonkeyPatch
 from omegaconf import OmegaConf
 
-from torchgeo.trainers import ClassificationTask, So2SatDataModule
+from torchgeo.trainers import (
+    BigEarthNetDataModule,
+    ClassificationTask,
+    MultiLabelClassificationTask,
+    So2SatDataModule,
+)
 
 from .test_utils import mocked_log
 
 
 @pytest.fixture(scope="module", params=[("rgb", 3), ("s2", 10)])
-def bands(request: SubRequest) -> Tuple[str, int]:
```

Comment:
Since these bands and datamodule fixtures are specific to the task being tested, let's move them into the class definition, then their names don't need to change. For example, `bands_so2sat` would be under the `TestClassificationTask` class and be called `bands`.